staff/daniela-aguilera/cv-daniela/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cv</title>
+</head>
+
+<body>
+    <section>
+        <header>
+            <h1>Marie Curie Cv</h1>
+            <img src="https://i.ytimg.com/vi/LTYJ1gYrzb8/maxresdefault.jpg" />
```

Comment:
@codingwithdani  "alt" attribute is highly recommended in all img tags. Please click on the link for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img .

Reply:
Done!